File: ws/nacato/u2/ws16Libraries/StoreApp/src/ec/edu/espe/store/view/Store.java
package ec.edu.espe.store.view;
import ec.edu.espe.store.model.Product;
import utils.Tax;

/**
 *
 * @author Angie Nacato, Error 404, @ESPE
 */
public class Store {

    public static void main(String[] args) {
        int id;
        String description;
        float price;
        float pvp;
        Product product;
        
        //TODO read from keyboard
        
        id=1;
        description="computer";
        price=100;
        
        pvp = price + Tax.computeTotal(price, 15F);
        
        product = new Product(id, description, price);
        System.out.println("product -->" + product);
        
        Product product2;
        
        id=2;
        description= "mouse";
        price = 1000;
        pvp = price + Tax.computeTotal(price, 15F);
        product2 = new Product(id, description, price, pvp);
        
        System.out.println("product 2 -->" + product2);
    }
}

File: hw/vizcaino/u2/hw09Libraries/hwLibrariesJavaScript/store.js
const Product = require('./model/product');
const Tax = require('./utils/tax');

let id = 1;
let description = "computer";
let price = 100;

let pvp = Tax.computeTotal(price, 15);

let product = new Product(id, description, price);

console.log("Product ---> " + product.toString());

id = 2;
description = "Mouse";
price = 1000;

pvp = Tax.computeTotal(price, 15);

let product2 = new Product(id, description, price, pvp);

console.log("Product2 ---> " + product2.toString());